dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-3.0/io.openliberty.jaxb-3.0.feature

```diff
@@ -23,4 +23,4 @@ Subsystem-Name: Jakarta XML Bindings 3.0
 -bundles=\
   io.openliberty.jaxb.3.0.internal.tools
 kind=beta
-edition=base
+edition=core
```

Comment:
You made this core because your feature depends on it.  I see that jpaContainer also depends on it.  I wonder if both your and the jpaContainer features should depend on the interface feature / bundle instead of the impl.  Then this feature could stay base.

Reply:
See the comment for this issue below.